SU-SWS/saa_victoria_subtheme
# Adapt 684  back to top

# NOT READY

# Summary
- fix the back to top accessibility issue (focus not switching)

# Review By (Date)
- 9/15

# Urgency
- medium

# Steps to Test

1. Pull down the branch
2. Clear the site cache
3. go to node/61  locally and click on a 'back to top' link and then immediately hit tab to see where the focus switched to. it should NOT be down by the back to top link you came from; it should switch up to near the top of the page.
4. you should see an id="associates--top-of-page" not a name="top", and the back to top links should all be linking to #associates--top-of-page not #top.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-684
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
